scripts/lib/CIME/check_lockedfiles.py

```diff
@@ -9,6 +9,8 @@
 from CIME.XML.env_batch import EnvBatch
 from CIME.utils import run_cmd_no_fail
 
+logger = logging.getLogger(__name__)
```

Comment:
Yes pylint is fine with it, I would like to leave it in place.

 Tools/code_checker check_lockedfiles.py
File scripts/lib/CIME/check_lockedfiles.py has no pylint problems